packages/dataviews/src/components/dataviews-item-actions/index.tsx

```diff
@@ -179,6 +185,13 @@ export function ActionsMenuGroup< Item >( {
 	);
 }
 
+function hasOnlyOneActionAndIsPrimary< Item >(
+	primaryActions: Action< Item >[],
+	actions: Action< Item >[]
+) {
+	return primaryActions.length === 1 && actions.length;
```

Comment:
Why not inlining the check?

Reply:
That could work as well. No big feelings either way. I suppose I thought it better to have it explain the  _why_.